### PR TITLE
feat: copy version, description and authors from pyproject

### PR DESCRIPTION
**Pull request by baszalmstra**
Copies the `version`, `description`, and `authors` from the pyproject if they are not explicitly defined in the `[tool.pixi.project]` section.

### CODE REVIEW

**Review by ruben-arts — APPROVED** [review]
I'm good with merging, but you sure you want to leave this TODO in?

File: crates/pixi_manifest/src/pyproject.rs [hunk 1]
```diff
@@ -67,9 +67,39 @@ impl From<PyProjectManifest> for ParsedManifest {
...
+                        (Some(name), Some(email)) => Some(format!("{} <{}>", name, email)),
+                        (Some(name), None) => Some(name.clone()),
+                        (None, Some(email)) => Some(email.clone()),
+                        (None, None) => None,
+                    })
+                    .collect()
+            });
+        }
+
+        // TODO: Add license, license-file, readme, homepage, repository, documentation,
```
[the remark this answers is not in the document]
Comment (tdejager) [reply crates/pixi_manifest/src/pyproject.rs:101]: You mean add it here?

**Review by tdejager — APPROVED** [review]
Other than what ruben mentioned, approved.

File: crates/pixi_manifest/src/pyproject.rs (hunk 1 above)
Comment (tdejager) [inline crates/pixi_manifest/src/pyproject.rs:101]:
```suggestion
        // TODO:  would be nice to add license, license-file, readme, homepage, repository, documentation,
        // regarding the above, the types are a bit different than we expect, so the conversion is not straightforward
        // we could change these types or we can convert. Let's decide when we make it.
```

**Pull request merged**